scanner/bin/dynamo.js

```diff
@@ -0,0 +1,190 @@
+const debug = require('debug')('Morar:database');
+const AWS   = require('aws-sdk');
+AWS.config.update({region: process.env.AWS_REGION || 'eu-west-1'});
+
+const Dynamo       = new AWS.DynamoDB();                // for basic table accesses
+const DynamoClient = new AWS.DynamoDB.DocumentClient(); // for metadata about a table
+
+function describeTable(table){
+	return new Promise( (resolve, reject) => {
+		if(table === undefined || table === null){
+			reject(`'table' argument is ${table}`);
+		} else {
+			Dynamo.describeTable({
+				TableName : table
+			}, (err, result) => {
+				if(err){
+					reject(err);
```

Comment:
Promise rejection handling?

Reply:
The error is handled in the app that calls the module, not in the module itself.